src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -868,45 +868,40 @@ public StringBuilder notifyHook(String url, String id, String action, String str
 			}
 
 			try {
-				response = sendPOST(url, variables);
+				sendPOST(url, variables, appSettings.getWebhookRetryCount());
 			} catch (Exception e) {
-				//Make Exception generi
 				logger.error(ExceptionUtils.getStackTrace(e));
 			}
 		}
-		return response;
+		return null;
```

Comment:
If you always return null, please change the return type to void.

Reply:
This method doesn't always return null; if successful, it returns the response. However, the return value is not utilized in the product code itself; it is only employed for unit testing purposes.

Reply:
>This method doesn't always return null; if successful, it returns the response
The read code here always return null. Please correct if I'm wrong. 

>However, the return value is not utilized in the product code itself; it is only employed for unit testing purposes.
If the real code always return null and it's ok, no need to make it return a value for the unit test. 

It creates ambiguity and makes it harder to maintain.

Reply:
converted sendPOST to void and edited unit test accordingly.
return value of sendPost has never used on real code anyway.
thanks for the review